src/sql/strong-oracle/create.sql

```diff
@@ -0,0 +1,332 @@
+BEGIN
```

Comment:
@scratch28 The stack overflow answer argues that one query is better than two, which is a small reason in this case, but so far I haven't heard any justifications for the other side, so I'll leave it as-is unless there is a justification to change.